### ISSUE TITLE
Rule 'unique-operation-name' requires 'parserOptions.operations' to be set and loaded. 

### ISSUE THREAD

**Issue by runar-rkmedia** [dotansimha/graphql-eslint#493]
**Describe the bug**

I am getting the error 

> Rule 'unique-operation-name' requires 'parserOptions.operations' to be set and loaded. 

However, I believe I have set the settings correctly according to the Readme.

**To Reproduce**

I am 

**Expected behavior**

**Environment:**

- OS: macOs Big Sur 11.4
- `@graphql-eslint/...`: v1.0.1 (sha512-+7nOx36ZgP57iL4tUbMTmILasqaL9FeYiH8G+/Co7mY871OmHNIS91izHaAZHh28emCE6dW/J1Wt9dhmag/OUw==)
- NodeJS: v14.15.1

**Additional context**

eslintrc.yaml
```yaml
---
extends: react-app
overrides:
  - files:
      - "*.graphql"
    parser: "@graphql-eslint/eslint-plugin"
    plugins:
      - "@graphql-eslint"
    parserOptions:
      operations:
        - "./src/**/*.graphql"
      schema: "./schema.graphql"
    rules:
      "@graphql-eslint/unique-operation-name": 2
```

Relevant stack-trace-error:
```
[Error - 1:43:34 PM] ESLint stack trace:
[Error - 1:43:34 PM] Error: Rule 'unique-operation-name' requires 'parserOptions.operations' to be set and loaded. See http://bit.ly/graphql-eslint-operations for more info
Occurred while linting /myproject/src/components/Approve/OrderApprovals.graphql:1
    at requireSiblingsOperations (/myproject/node_modules/@graphql-eslint/eslint-plugin/index.cjs.js:24:15)
    at OperationDefinition (/myproject/node_modules/@graphql-eslint/eslint-plugin/index.cjs.js:1559:34)
    at /myproject/node_modules/eslint/lib/linter/safe-emitter.js:45:58
    at Array.forEach (<anonymous>)
    at Object.emit (/myproject/node_modules/eslint/lib/linter/safe-emitter.js:45:38)
    at NodeEventGenerator.applySelector (/myproject/node_modules/eslint/lib/linter/node-event-generator.js:293:26)
    at NodeEventGenerator.applySelectors (/myproject/node_modules/eslint/lib/linter/node-event-generator.js:322:22)
    at NodeEventGenerator.enterNode (/myproject/node_modules/eslint/lib/linter/node-event-generator.js:336:14)
    at CodePathAnalyzer.enterNode (/myproject/node_modules/eslint/lib/linter/code-path-analysis/code-path-analyzer.js:711:23)
    at /myproject/node_modules/eslint/lib/linter/linter.js:956:32
```

**Comment by ardatan** [comment]
Could you share a reproduction on CodeSandbox or somewhere else?

**Comment by runar-rkmedia** [comment]
I tried to create a MVE here, but it seems to work correctly there. I'll try to look a bit more into reproducing this.

**Comment by runar-rkmedia** [comment]
Turns out, there was a hidden `.graphql-config` file that I did not know about in the project. I think it is outdated.

For now, I set `skipGraphQLConfig: true` as suggested in [./docs/parser-options.md](https://github.com/dotansimha/graphql-eslint/blob/master/docs/parser-options.md)

I think we can close this now.
